jerry-core/parser/js/js-parser.c

```diff
@@ -1450,7 +1450,7 @@ parser_post_processing (parser_context_t *context_p) /**< context */
   needs_uint16_arguments = false;
   total_size = sizeof (cbc_uint8_arguments_t);
 
-  if ((context_p->register_count + context_p->stack_limit) > CBC_MAXIMUM_BYTE_VALUE
+  if (context_p->stack_limit > CBC_MAXIMUM_BYTE_VALUE
```

Comment:
I just realized we should check the register_count as well:

    || context_p->register_count > CBC_MAXIMUM_BYTE_VALUE

It is true that PARSER_MAXIMUM_NUMBER_OF_REGISTERS is 256 at this moment, but it can be changed.